lib/tasks/maintenance/cached/maps.rake

```diff
@@ -103,18 +103,18 @@ namespace :tw do
             h = CachedMapItem.cached_map_name_hierarchy(o.geographic_item_id)
 
             z = CachedMapItem.where(geographic_item_id: o.geographic_item_id)
-              .where.not(untranslated: true)
+              .where('untranslated IS NULL OR untranslated = FALSE')
```

Comment:
Per change does `.where(untranslated: [nil, false])` provide similar behavior?

```
CachedMapItem.where(geographic_item_id: 3).where(untranslated: [nil, false]).to_sql
 => "SELECT \"cached_map_items\".* FROM \"cached_map_items\" WHERE \"cached_map_items\".\"geographic_item_id\" = 3 AND (\"cached_map_items\".\"untranslated\" = FALSE OR \"cached_map_items\".\"untranslated\" IS NULL)"
 ```

Reply:
Yah, that should be the same, thanks - I'll change.